src/gear/history_translator.cc

```diff
@@ -0,0 +1,66 @@
+//
+// Copyright RIME Developers
+// Distributed under the BSD License
+//
+// 2016-09-08 osfans <waxaca@163.com>
+//
+
+#include <rime/candidate.h>
+#include <rime/context.h>
+#include <rime/engine.h>
+#include <rime/schema.h>
+#include <rime/ticket.h>
+#include <rime/translation.h>
+#include <rime/gear/history_translator.h>
+
+namespace rime {
+
+HistoryTranslator::HistoryTranslator(const Ticket& ticket)
+    : Translator(ticket),
+      tag_("abc"),
+      input_("ls"),
```

Comment:
Remove the default value.
The keyword is schema-specific and need to be explicitly defined.